systemtest/010-inspect.bats

```diff
@@ -65,21 +65,27 @@ END_EXPECT
 }
 
 @test "inspect: env" {
-    remote_image=docker://docker.io/fedora:latest
+    remote_image=docker://quay.io/libpod/fedora:31
     run_skopeo inspect $remote_image
     inspect_remote=$output
 
     # Simple check on 'inspect' output with environment variables.
     #    1) Get remote image values of environment variables (the value of 'Env')
     #    2) Confirm substring in check_array and the value of 'Env' match.
-    check_array=(PATH=.* )
+    check_array=(FGC=f31 DISTTAG=f31container)
     remote=$(jq '.Env[]' <<<"$inspect_remote")
     for substr in ${check_array[@]}; do
         expect_output --from="$remote" --substring "$substr"
     done
 }
 
 @test "inspect: image manifest list w/ diff platform" {
+    # But how? What is the point of this test? Can we just remove the
+    # entire test and do a simple inspect of quay.io/libpod/busybox:latest?
+    # That has a ton of arches:
+    #  inspect... | jq -r '.manifests | .[] | .platform.architecture' | sort -u
+    skip "FIXME FIXME FIXME THIS TEST NEEDS TO BE REWRITTEN"
+
```

Comment:
@edsantiago if you have not yet, could you make an issue against Skopeo so we don't forget to circle back to this?

Reply:
>could you make an issue against Skopeo so we don't forget to circle back to this?

@TomSweeneyRedHat final followup: thanks to valuable feedback from @mtrmac , the test is reenabled in a form that relies only on quay, and should pass on all RHEL builds that I know of. 

One important caveat is that the test will fail if we switch to running CI on AmigaOS. I choose to defer that problem until such date.